inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/postgres-cdc/src/main/java/org/apache/inlong/sort/cdc/postgres/source/config/PostgresSourceConfig.java

```diff
@@ -100,7 +101,7 @@ public RelationalTableFilters getTableFilters() {
 
     @Override
     public List<String> getMetricLabelList() {
-        return Arrays.asList(AbstractSourceInfo.DATABASE_NAME_KEY,
+        return Arrays.asList(DATABASE_NAME_KEY,
```

Comment:
`cdc-base` module already has `DATABASE_NAME` constant, so you can import it, not define it by yourself.

<img width="696" alt="image" src="https://github.com/apache/inlong/assets/111486498/5492f9c2-5b83-459a-9ddb-32e39e74cdf2">

you also refer Oracle CDC. 

<img width="1040" alt="image" src="https://github.com/apache/inlong/assets/111486498/feaa5bec-9a8e-4614-a262-2fe7cc0fb9f5">